src/installer/pkg/sfx/Microsoft.NETCore.App/ReadyToRun.targets

```diff
@@ -46,10 +47,16 @@
   </PropertyGroup>
 
   <Target Name="PatchDotnetRootBeforeRunningCrossgen2" BeforeTargets="_CreateR2RImages">
+    <ReadLinesFromFile File="$(ArtifactsDir)/toolset/sdk.txt">
+      <Output TaskParameter="Lines" ItemName="DotNetRoot"/>
```

Comment:
```suggestion
      <Output TaskParameter="Lines" PropertyName="DOTNET_ROOT"/>
```

Reply:
You may assign directly to a property and remove `<DOTNET_ROOT>@(DotNetRoot)</DOTNET_ROOT>`.

Reply:
Thanks, will do. Sadly this doesn't change the fact that it still doesn't work, either the file doesn't exist or the script is unable to read it. I'm waiting on the Linux runs to see whether it's just a problem of the directory separator formatting, it's actually pretty hard for me to test locally as I naturally have the latest dogfooding SDK installed, otherwise I basically wouldn't be able to debug anything in the VS.